### DISCUSSION TITLE
Auto start Maui application on OS boot

### DISCUSSION THREAD

**Discussion by SebastianKleu** [dotnet/maui#7032] [Q&A] [answered]
Hi, can I auto start a Maui application? I ported code I used for a Xamarin app but the app does not start up when OS restarted. I used a BroadcastReceiver and included my receiver configuration within AndroidManifest.xml file.

I am using Android 10 OS.

**Comment by angelru** [comment, marked as answer]
You must register the Broadcast in an Implicit way

https://stackoverflow.com/questions/71235614/android-app-not-starting-on-boot-completed-with-android-xamarin
https://stackoverflow.com/questions/49403069/xamarin-android-receiver-on-boot-completed-error

**Comment by SebastianKleu** [comment]
Thank you! 
The second post sorted me out. 
https://stackoverflow.com/questions/49403069/xamarin-android-receiver-on-boot-completed-error

**Reply by angelru** [reply]
glad to hear it! mark the question as solved/answered

**Comment by HuaFangYun** [comment]
Would like to ask maui android boot to achieve it? Do you have a demo?

**Comment by dflyers** [comment]
nice to have this feature for Android.
